seldon-core-operator/rockcraft.yaml

```diff
@@ -18,9 +18,9 @@ services:
   manager:
     override: replace
     summary: "seldon-core service"
-    startup: enabled
+    startup: disabled
```

Comment:
What would be the reason to disable it?

Reply:
I commented on Kubeflow couple of days ago.
#kf-help For all ROCKers out there.... I spent a lot of time on this.....
In order for Charm to properly start ROCK-based container it MUST be startup should be disabled in services in `rockcraft.yaml`.
```
services:
  manager:
    override: replace
    summary: "seldon-core service"
    startup: disabled #<<<<<<<<<<<<< MUST BE DISABLED
    user: ubuntu
    command: "/manager --enable-leader-election --webhook-port 4443"
```
When startup is enabled, executable/command is started by pebble. When charm comes in, it tries to do the same thing and fails, it looks like it launches another process using command specified in charm's services (same as in ROCK). Then behaviour will differ from charm to charm, but in case of Seldon second copy fails due to port 8080  being taken by first copy.
It smells to me like a bug in Pebble though or I don't understand something.

Reply:
From a conversation with the [pebble team](https://chat.charmhub.io/charmhub/pl/paxkjackabymbx8gn8qgchuzpe) and the [pebble readme](https://github.com/canonical/pebble/#readme), it seems like the issue is rather in the charm's pebble layer configuration. If we were to use this rock in seldon-core-operator, we'd have to change [the service name in the pebble layer](https://github.com/canonical/seldon-core-operator/blob/main/src/charm.py#L240) to match that of the rockcraft file to avoid running the service twice; the other option is to play around with pebble layer configurations, for instance, using different `override` policies to use what is already defined in the rockcraft file.

Reply:
Re-testing.

Reply:
Updated PR.

Reply:
Yes. I put it back as enabled.